apps/partner2/client/view.coffee

```diff
@@ -149,7 +150,7 @@ module.exports = class PartnerView extends Backbone.View
     criteria =
       overview: => true
       shows: => @partner.get('displayable_shows_count') > 0
-      artists: => @partner.get('partner_artists_count') > 0
+      artists: => @partner.get('partner_artists_count') > 0 and @partner.get('display_artists_section')
```

Comment:
As noted - this works as expected 👍  The property controls artists display on the overview page. Maybe language update might be in order in volt.